app/models/experience.rb

```diff
@@ -48,7 +48,8 @@ class Experience < ActiveRecord::Base
   validates :link,        url: true, allow_blank: true
 
   def title
-    self.description.present? ? self.description.truncate(50) : nil
+    # Since we allow RTF in the description, we should strip any HTML from the title
+    self.description.present? ? ActionController::Base.helpers.strip_tags(self.description).truncate(30) : nil
```

Comment:
Any reason you went down to 30 from 50 chars? Otherwise this seems reasonable to me.